staging/src/k8s.io/kubectl/pkg/cmd/auth/cani.go

```diff
@@ -105,7 +105,7 @@ var (
 		# List all allowed actions in namespace "foo"
 		kubectl auth can-i --list --namespace=foo`)
 
-	resourceVerbs       = sets.NewString("get", "list", "watch", "create", "update", "patch", "delete", "deletecollection", "use", "bind", "impersonate", "*", "approve")
+	resourceVerbs       = sets.NewString("get", "list", "watch", "create", "update", "patch", "delete", "deletecollection", "use", "bind", "impersonate", "*", "approve", "sign", "escalate")
```

Comment:
Do we want to add `attest` even though cluster trust bundles are alpha?

Reply:
I can feature-flag add it :)

Reply:
Ah, can't import `k8s.io/kubernetes` in `staging/` packages